src/Language/Haskell/Liquid/Types/PrettyPrint.hs

```diff
@@ -62,6 +72,10 @@ import           Language.Haskell.Liquid.Types.Types
 import           Prelude                          hiding (error)
 import           Text.PrettyPrint.HughesPJ        hiding ((<>))
 
+data Filter = StringFilter String
+            | AnyFilter
```

Comment:
This is a good place to explain what a Filter is, where they come from, and how they are used.

Reply:
Changed.